source/stable-release_iot/feature-delta-iot.rst

```diff
@@ -0,0 +1,76 @@
+    :local:
+* 100% Google Conformance on Comet Lake(CML),Tiger Lake(TGL) & Elkhart Lake(EHL) with Production ready
+* Supporting Comet Lake(CML),Tiger Lake(TGL) & Elkhart Lake(EHL) on single baseline and Single Image
+* Elkhart Lake(EHL) Platform supported on Celadon with 100% compliance
+* IoT configuration support GVT-d mode with 100% conformance
+* IoT configuration support GVT-g with no conformance
+
+Real-time Clock (RTC) & Alarm Virtualization
+********************************************
+
+Celadon in Virtual Machine (CiV) project may requires operations on the RTC device, such as changes to current time and setups of alarm, to be synced to host and persistent across reboots. RTC device is purely emulated by QEMU and its state will be lost after exit of QEMU process, furthermore, guest RTC alarm is unable to wake up the guest when host is also suspended, which is the case in when using --host-pm-control option in CiV. 
+
+The solution is to modify the emulated RTC device so that it will emit QMP events upon certain operations, and use a QMP client running on host to receive the QMP events and sync the operations to host accordingly.
+
+.. figure:: images/rtc.png
+        :width: 500px
+        :align: center
+
+Figure 1 : Architecture Diagram of RTC Time Virtualization
+
+"Whenever the guest RTC alarm time is set, QEMU will emit an RTC_CHANGE event to QMP client, and the QMP client can then setup a host RTC alarm accordingly".
+
+.. figure:: images/alarm.png
+        :width: 500px
+        :align: center
+
+Figure 2 : Architecture Diagram of RTC Alarm Virtualization
+
+Whenever the expire time of guest RTC alarm changes, QEMU will emit an RTC_ALARM event to QMP client, and the QMP client can then setup a host RTC alarm accordingly. In the case of host suspension, the host alarm will wake up the host and notify the QMP client, then QMP client will send an rtc-refresh-timer command to QEMU, to ensure that QEMU will wake up the guest.
+
+Secure Data Erase
+*****************
+
+In Celadon, all partitions are inside a single guest virtual disk image file on the host with the disk controller emulated by QEMU. The underlying actual storage hardware technology be it magnetic or solid state is transparent to the Android VM.
+
+As such, due to emulation, secure partition erase would be fall back to “byte-by-byte overwrite” to the virtual partition. This “byte-by-byte overwrite” operation is not solid state storage friendly.
+
+To provide secure data erase of data partition, following enhancement is needed in CiV:
+
+Addition of secure data erase to make use of storage controller hardware secure erase feature, when it is supported in hardware to erase data partition and only finally fall back to byte-by-byte override 0 when not supported during Android recovery/wipe data process.
+
+#. Guest virtual disk image file enhancement
+
+   In Celadon CiV, all Android required partitions are resided in the same guest virtual disk image file in host.
```

Comment:
what is Celadon CiV? CiV is Celadon in VM.

Reply:
Updated as per the Suggestion